plugins/wme-sitebuilder/wme-sitebuilder/Plugin.php

```diff
@@ -133,4 +136,28 @@ public function registerModule( $module ) {
 
 		return $this;
 	}
+
+	/**
+	 * Add the new admin color scheme.
+	 */
+	public function add_admin_color_scheme() {
+		return wp_admin_css_color(
```

Comment:
```suggestion
		wp_admin_css_color(
```
`wp_admin_css_color` is a void function and doesn't return anything.